README.md

```diff
@@ -1,2 +1,242 @@
+It can be performed multiple times in a cycle of:  🗣❌👎gathering feedback on
+your designs from users -> adapting your designs in light of user feedback 🤔 ✏️
+-> getting feedback on your amended designs to see if they now meet the users'
+needs 🗣✅👍. Once you are satisfied the users needs have been met then you can
+begin to build the agreed designs. 🏗 👷‍♀️ 👷
+
+## How?
+
+There are many approaches to user experience testing that you may choose to use
+based on your product, stage of product development, budget 💵, client base 👥 etc.:
+
+The first stage of user testing you may wish to conduct is known as the discovery
+phase, for information on this stage see: https://www.gov.uk/service-manual/agile-delivery/how-the-discovery-phase-works
+
+To perform usability user testing you may use one/some of the following methods:
+
+* interviews
+* Surveys/ questionnaires
+* Usage pattern analysis using "analytics" data. For more on this see:
+https://github.com/dwyl/learn-analytics 📊
+* Screen recordings #1 🎥🖥
+* App usage/demo video capture
+
+Ensure that when you conduct tests you are testing them across devices so that
+your data reflects all of your users experiences e.g. mobile 📱, tablet and
+desktop 💻 .
+
+### How to conduct interviews
+1. Create a **script** to run through with your testers 📃
+2. Decide **who** you will interview 👥
+3. Find some**where** appropriate to conduct your interviews 🏢
+4. Decide on a **time/date** to perform your research 📆 🕐
+5. Decide **who** in the team will **conduct the interviews** 👩‍💼👨‍💼
+6. Decide **how many** interviews you wish to do
+6. Decide how you will **record the findings** of your interviews. 📝🔉🎥
+
+**1. Writing a script 🖊**
+A script is a helpful way of ensuring interviews are delivered in a consistent
+way (if different team members conduct them) and that the interviewer can feel
+relaxed ☺️ because they know what they've got to say. It's important that scripts
+aren't read like an autocue 🤖 as you want the tester to feel relaxed too, so
+making them more like a normal conversation is beneficial. However if you are
+anxious 😰 about missing something out, one way to deal with this is to tell the
+participant 'I'm going to read through this part of the process to make sure I
+don't leave anything out'. By telling them what you are doing and why it can
+help alleviate worries they may have constructed otherwise, you're telling them
+that you're reading it so they benefit from all of the information, not because
+this is a formal setting and you're avoiding their eye contact 👀😊! So make sure
+the team familiarises themselves with the script before they start. The script
+doesn't need to be followed word for word but team members should bare in mind
+that certain word choices are important in order to not influence the testers
+responses. Here is a useful script outline that might be of use for some
+projects: https://docs.google.com/forms/d/e/1FAIpQLSfM2Uaje8gpBde-RqU01DlxrGUEsWZrjiy8yTKmYaeJYAZUuw/viewform
```

Comment:
https://docs.google.com/forms/d/e/1FAIpQLSfM2Uaje8gpBde-RqU01DlxrGUEsWZrjiy8yTKmYaeJYAZUuw/viewform
![image](https://user-images.githubusercontent.com/194400/53006834-f16aed00-342d-11e9-8a4c-3389c93fd7f3.png)
Worked on the second attempt:
![image](https://user-images.githubusercontent.com/194400/53006924-25dea900-342e-11e9-9d14-60b728797939.png)

Structure of the survey could be improved (_too many questions before getting to the point_), but is a good starting point.

Reply:
I've added this note to the script:
> The script came from this 5 min Google tutorial: https://www.youtube.com/watch?v=0YL0xoSmyZI.
Remember though to always tailor your script to what you are testing e.g. you
may not need as many introductory questions as this one depending on what you
are aiming to find out from your testing at this stage.